setup.py

```diff
@@ -25,7 +25,7 @@
     include_package_data=True,
     install_requires=[
         'OpenFisca-Core >= 43, < 44',
-        'OpenFisca-France >= 169.0.0, < 171.0.0',
+        'OpenFisca-France >= 171.0.0, < 173.0.0',
```

Comment:
Qu'est-ce qui ne convenait pas pour les versions en dessous de 171 ? Je n'ai pas trouvé de raison d'y toucher

```suggestion
        'OpenFisca-France >= 169.0.0, < 173.0.0',
```

Reply:
> Qu'est-ce qui ne convenait pas pour les versions en dessous de 171 ? Je n'ai pas trouvé de raison d'y toucher

Non effectivement je n'avais pas vraiment de raison autre que c'était la version maximale de la précédente. J'ai suivi la recommandation.